roles/openshift_logging/tasks/install_logging.yaml

```diff
@@ -78,6 +78,7 @@
     openshift_logging_elasticsearch_nodeselector: "{{ openshift_logging_es_nodeselector if item.0.nodeSelector | default(None) is none else item.0.nodeSelector }}"
     openshift_logging_elasticsearch_storage_group: "{{ [openshift_logging_es_storage_group] if item.0.storageGroups | default([]) | length == 0 else item.0.storageGroups }}"
     _es_containers: "{{item.0.containers}}"
+    _es_configmap: "{{ openshift_logging_facts | walk('elasticsearch#configmaps#logging-elasticsearch#elasticsearch.yml', '{}', delimiter='#') | from_yaml }}"
```

Comment:
I believe this can be handled without adding these extra functions.
I think you can do this:
```
  _es_configmap: "{{ (openshift_logging_facts.elasticsearch.configmaps.logging-elasticsearch['elasticsearch.yml'] | default('{}')) | from_yaml }}"
```
Or using our existing modules:
```
  - oc_configmap:
      state: list
      namespace: logging
      name: logging-elasticsearch
    register: logout

  - yedit:
      content: "{{ logout.results.results[0] | to_yaml }}"
      key: data#elasticsearch.yml
      separator: '#'
      state: list
    register: yout

  - debug:
      msg: "{{ yout.result | from_yaml }}"
```